p2p/sync.go

```diff
@@ -435,7 +434,7 @@ func (s *syncService) genHeadersAndSigs(ctx context.Context, blockNumber uint64)
 
 type specClasses struct {
 	number  uint64
-	classes []*spec.Class
+	classes []core.Class
```

Comment:
changing this to core class is misleading since the spec classes name was not changed